src/course/en/course.json

```diff
@@ -1,7 +1,7 @@
 {
     "_id": "course",
     "_type": "course",
-    "_htmlClassName": "",
+    "_htmlClasses": "",
```

Comment:
I think you added `"_classes": "",` here (as well) in your v4 PR

Reply:
ah and of course you shouldn't need to here as well cause the v4 master branch should get merged into this one sorry

Reply:
yes I did; but I left that in https://github.com/adaptlearning/adapt_framework/pull/2552. I did not duplicate those additions here.